app/src/main/java/com/vultisig/wallet/presenter/import_file/ImportFileScreen.kt

```diff
@@ -123,7 +123,7 @@ private fun ImportFileScreen(
                 .padding(
                     vertical = 16.dp,
                 ),
-            onClick = ClickOnce(onContinue),
```

Comment:
The reason for the change is that the MultiColorButton(its parent) handles ClickOnce.